aws_lambda_powertools/logging/logger.py

```diff
@@ -583,6 +583,12 @@ def debug(
     def append_keys(self, **additional_keys: object) -> None:
         self.registered_formatter.append_keys(**additional_keys)
 
+    def get_current_keys(self) -> Dict[str, Any]:
+        if isinstance(self.registered_formatter, LambdaPowertoolsFormatter):
+            return self.registered_formatter.get_current_keys()
+
+        return {}
```

Comment:
We can now simplify this:

```suggestion
        return self.registered_formatter.get_current_keys()
```